src/scenes/game_view.cpp

```diff
@@ -121,26 +34,62 @@ class GameView : public Scene {
     };
     EnumArray<PauseButton, raylib::Rectangle> m_pause_button_rects;
     EnumArray<PauseButton, bool> m_pause_button_pressed;
-    bool should_draw_pause = false;
+    bool m_should_draw_pause = false;
+
     /* misc */
-    /* ... */
+    std::vector<std::filesystem::path> m_card_image_paths;
 
-public:
+protected:
     void on_window_attach() override {
-        m_player_hand.recalculate_card_rects(m_window->GetWidth(), m_window->GetHeight());
-        const raylib::Vector2 board_offset = {(m_window->GetWidth() - board_width) / 2.0f, 10.0f};
-        m_board_rect = raylib::Rectangle(board_offset.x, board_offset.y, board_width, board_height);
-        raylib::Image board_image;
-        try {
-            board_image.Load(board_image_path);
-            board_image.Resize(m_board_rect.GetSize().x, m_board_rect.GetSize().y);
-        } catch (const raylib::RaylibException &) {
-            board_image = raylib::Image::Color(
-                m_board_rect.GetSize().x, m_board_rect.GetSize().y, raylib::Color::RayWhite()
-            );
+        setup_background();
+        setup_pause_menu();
+        setup_hand();
+        m_board.setup(m_window, &m_player_hand);
+    }
+
+public:
+    explicit GameView() {
+        GuiLoadStyle("bin/gui_styles/meow.rgs");
+        GuiSetFont(LoadFont("bin/fonts/mono.ttf"));
+        GuiSetStyle(DEFAULT, TEXT_SIZE, 16);
+        for (const auto &entry : std::filesystem::directory_iterator("bin/imgs/cards/original/")) {
```

Comment:
since you have lots of hard-coded filepaths, it would make sense to create extra translation unit with all the string literals used in the project. It will make project easier to maintain because every path changes will be done in a separate translation unit while the logic itself will stay the same